### PR TITLE
Use rcutils to log

### PR DESCRIPTION
**Pull request by ahcorde**
Related with this issue https://github.com/ros2/urdf/issues/16

### CODE REVIEW

File: urdf/CMakeLists.txt [hunk 1]
```diff
@@ -118,6 +120,7 @@ ament_export_libraries(${PROJECT_NAME})
 ament_export_targets(${PROJECT_NAME})
 
 ament_export_dependencies(pluginlib)
+ament_export_dependencies(rcutils)
```
Comment (clalancette) [inline urdf/CMakeLists.txt]: We don't need to export a dependency on `pluginlib` or `rcutils` here, as they are both PRIVATE dependencies.

```suggestion
```

File: urdf/package.xml [hunk 2]
```diff
@@ -38,6 +40,7 @@
   <exec_depend>urdfdom_headers</exec_depend>
 
   <build_export_depend>pluginlib</build_export_depend>
+  <build_export_depend>rcutils</build_export_depend>
```
Comment (clalancette) [inline urdf/package.xml]: Same here; we don't need to `build_export_depend` here, since they are PRIVATE dependencies:

```suggestion
```

**Review by clalancette — CHANGES_REQUESTED** [review]

File: urdf/package.xml (hunk 2 above)
Comment (ahcorde) [reply urdf/package.xml]: https://github.com/ros2/urdf/pull/37/commits/e9419ebdf3a3fe600e587a0e437f66b56756ee74

File: urdf/CMakeLists.txt (hunk 1 above)
Comment (ahcorde) [reply urdf/CMakeLists.txt]: https://github.com/ros2/urdf/pull/37/commits/e9419ebdf3a3fe600e587a0e437f66b56756ee74

**Comment by ahcorde** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/21719/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/15994/)
* Linux-rhel [Build Status](http://ci.ros2.org/job/ci_linux-rhel/1224/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/22422/)

**Review by clalancette — APPROVED** [review]

**Pull request merged**